system/t07_serve/__init__.py

```diff
@@ -8,8 +8,43 @@
 import subprocess
 import shlex
 import time
+import errno
 
 from lib import BaseTest
+from socket import error as socket_error
+
+class RootDirInaccessible(BaseTest):
+    """
+    serve command aborts if rootDir is inaccessible
+    """
+    fixtureDB = False
+    fixturePool = False
+
+    configOverride = {
+        "rootDir": "/root" # any directory that exists but is not writable
+    }
+    runCmd = "aptly serve -listen=127.0.0.1:8765"
+
+    def run(self):
+        try:
+            proc = subprocess.Popen(shlex.split(self.runCmd), stderr=subprocess.STDOUT, stdout=subprocess.PIPE, bufsize=0)
```

Comment:
I haven't tested that, but do we need this logic here? `aptly serve` should fail immediately as `rootDir` is not writeable, so it should be more or less like `Serve2Test` below